src/stream-to-buffer.iced

```diff
@@ -0,0 +1,14 @@
+stream = require('stream')
+
+exports.StreamToBuffer = class StreamToBuffer extends stream.Transform
+
+  constructor : (options) ->
+    @bufs = []
+    super(options)
+
+  _write : (chunk, encoding, cb) ->
+    @bufs.push(chunk)
```

Comment:
Results in individual `await` times being longer. No real reason but I suspect this is *slightly* more time-efficient.